substrate/frame/safe-mode/src/lib.rs

```diff
@@ -15,6 +15,48 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! # Safe Mode
```

Comment:
Can you please put a warning on top that it is not yet audited? Something like:  

```
# WARNING

NOT YET AUDITED. DO NOT USE IN PRODUCTION.
```

Also for the tx-pause pallet.